uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java

```diff
@@ -0,0 +1,324 @@
+		propertiesView.activate();
+		propertiesView.setProperty(PropertyType.COMBO, "Language", "simple");
+		propertiesView.setProperty(PropertyType.TEXT, "Expression", MESSAGE);
+		editor.save();
+
+		Pattern p = Pattern.compile("^" + MESSAGE);
+		camelExample.getRunner().registerPattern(p);
+		camelExample.getRunner().waitForOutputWithPattern(p);
+
+		assertTrue("Message 'Hello world' was not changed to '" + MESSAGE + "' correctly",
+				camelExample.getRunner().getOutput().toString().contains(MESSAGE));
+
+		LogChecker.assertNoFuseError();
+	}
+
+	/**
+	 * <p>
+	 * Test tries to start tracing route via Jolokia connection
+	 * </p>
+	 * <ul>
+	 * <li>Create a new connection in JMX Navigator</li>
+	 * <li>New connection... --> <i>Jolokia Connection</i></li>
+	 * <li>Provide the name you want</li>
+	 * <li>Use Jolokia URL from Console View (by default <i>http://127.0.0.1:8778/jolokia</i>)</li>
+	 * <li>Select <i>Do NOT verify SSL Certificates</i></li>
+	 * <li>Click Finish</li>
+	 * <li>Connect to new <i>Jolokia</i> connection</li>
+	 * <li>Select route, open context menu and click on <i>Start Tracing</i></li>
+	 * <li>Wait short time period and then select route, open context menu and click on <i>Stop Tracing Context</i></li>
+	 * <li>Open <p>Message view</p></li>
+	 * <li>Verifies Message view contains <p>Hello World</p> messages</li>
+	 * <li>Verify Error log does not contains any <i>Fuse</i> errors</li>
+	 * </ul>
+	 */
+	@Test
+	public void testTracingRoute() {
+
+		TreeItem item = jmxView.getNode("User-Defined Connections", conf.getName(), "Camel", "SampleCamel");
+		item.select();
+		new ContextMenuItem(item, "Start Tracing").select();
+		AbstractWait.sleep(TimePeriod.MEDIUM);
+		jmxView.activate();
+		item.select();
+		new ContextMenuItem(item, "Stop Tracing Context").select();
+		
+		messageView.open();
+		assertTrue("Message view does not contains 'Hello World' message",
+				messageView.containsMessageBody("Hello World"));
+		LogChecker.assertNoFuseError();
+	}
+
+	/**
+	 * <p>
+	 * Test tries to start tracing route via Jolokia connection
+	 * </p>
+	 * <ul>
+	 * <li>Create a new connection in JMX Navigator</li>
+	 * <li>New connection... --> <i>Jolokia Connection</i></li>
+	 * <li>Provide the name you want</li>
+	 * <li>Use Jolokia URL from Console View (by default <i>http://127.0.0.1:8778/jolokia</i>)</li>
+	 * <li>Select <i>Do NOT verify SSL Certificates</i></li>
+	 * <li>Click Finish</li>
+	 * <li>Connect to new <i>Jolokia</i> connection</li>
+	 * <li>Set breakpoint to <p>Transform transform1</p> component and wait until debugger is enabled</li>
+	 * <li>Verify route was stopped</li>
+	 * <li>Click resume (route should have again stops on breakpoint)</li>
+	 * <li>Remove breakpoint</li>
+	 * <li>Click resume (route should have continue)</li>
+	 * <li>Verify Error log does not contains any <i>Fuse</i> errors</li>
+	 * </ul>
+	 */
+	@Test
+	public void testDebuggingRoute() {
+
+		activateRoute();
```

Comment:
On the first sight, it was a bit confusing for me. What about changing the name of the method to something more descriptive like _activateRouteEditing_